docs/features/software-templates/installation.md

```diff
@@ -238,10 +243,24 @@ scaffolder:
           env: SCAFFOLDER_GITLAB_PRIVATE_TOKEN
 ```
 
-You can configure who can see the new repositories that the scaffolder creates
-by specifying `visibility` option. Valid options are `public`, `private` and
-`internal`. `internal` options is for GitHub Enterprise clients, which means
-public within the organization.
+#### Azure DevOps
+
+For Azure DevOps we support both the preparer and publisher stage with the
+configuration of a private access token (PAT) when needed. For the publisher
+it's also required to define the organization that you want to create the
+repository in.
+
+```yaml
+scaffolder:
+  azure:
+    api:
```

Comment:
We're not really expecting to keep the extra `api` distinction afaik, prolly better to just make it `scaffolder.azure.organization` etc

Reply:
I would be happy to. I'm not really sure where the api part came from but I stuck with it since the other ones have it.